src/NServiceBus.AcceptanceTests/EndpointTemplates/DefaultServer.cs

```diff
@@ -56,7 +57,7 @@ public DefaultServer(List<Type> typesToInclude)
 
             if (serializer != null)
             {
-                builder.UseSerialization(Type.GetType(serializer));
+                builder.UseSerialization((SerializationDefinition)Activator.CreateInstance(Type.GetType(serializer, true)));
```

Comment:
This is ugly but I see there is no other way around because you want to deprecate

` public static void UseSerialization(this NServiceBus.BusConfiguration config, System.Type serializerType)`

I first thought we can repurpose the above method to:

 `public static void UseSerialization(this NServiceBus.BusConfiguration config, System.Type serializerDefinitionType)`

But that would be strange